### PR TITLE
onboarding page_first upload

### PR DESCRIPTION
**Pull request by Tinjomar**
#3 

https://gyazo.com/4b56dd4dd236a6df70c65a3ecf77cc45

### CODE REVIEW

File: code_onboarding_page.dart [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+import 'package:intro_slider/intro_slider.dart';
+import 'package:intro_slider/slide_object.dart';
+
+class OnBoardingPage extends StatefulWidget {
+  @override
+  _OnBoardingPageState createState() => _OnBoardingPageState();
+}
+
+class _OnBoardingPageState extends State<OnBoardingPage> {
+  List<Slide> slides = new List();
```
Comment (matejthetree) [inline code_onboarding_page.dart]: `= [];`

Comment (matejthetree) [reply code_onboarding_page.dart]: we dont use new

File: code_onboarding_page.dart [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+  @override
+  void initState() {
+    super.initState();
+
+    slides.add(
+      new Slide(
```
Comment (matejthetree) [inline code_onboarding_page.dart]: no `new`

File: code_onboarding_page.dart [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+              height: 310,
+              decoration: BoxDecoration(
+                  color: Colors.white,
+                  image: DecorationImage(
+                      image: NetworkImage('https://placekitten.com/200/300'))))),
+    );
+  }
+
+  void onDonePress() {  // Done button is START button
+    // TODO: go to next screen
```
Comment (matejthetree) [inline code_onboarding_page.dart]: navigate to home page

File: code_onboarding_page.dart [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+  }
+
+
+  @override
+  Widget build(BuildContext context) {
+    return IntroSlider(
+      slides: this.slides,
+      onDonePress: this.onDonePress,
+        nameNextBtn: ('SKIP'),
```
Comment (matejthetree) [inline code_onboarding_page.dart]: next

File: lib/main.dart [hunk 5]
```diff
@@ -10,12 +11,8 @@ void main() {
 class YTT extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    return MultiProvider(
```
Comment (matejthetree) [inline lib/main.dart]: nije dobro, ne brisati

File: lib/main.dart [hunk 6]
```diff
@@ -10,12 +11,8 @@ void main() {
...
-      providers: providers,
-      child: MaterialApp(
-        routes: routes,
-        initialRoute: initialRoute,
-      ),
+    return MaterialApp(
+      home: OnBoardingPage (),
```
Comment (matejthetree) [inline lib/main.dart]: nije dobro, moras koristiti test rute, @Noamagnus moze objasnit

Comment (matejthetree) [reply lib/main.dart]: ne treba test ruta, trebas napravit onboarding routu, jer ce ona biti u aplikaciji

File: code_onboarding_page.dart [hunk 7]
```diff
@@ -0,0 +1,76 @@
+
```
Comment (matejthetree) [inline code_onboarding_page.dart]: samo `onboarding_page.dart`

File: lib/ui/page/code_onboarding_page.dart [hunk 8]
```diff
@@ -0,0 +1,76 @@
+
```
Comment (matejthetree) [inline lib/ui/page/code_onboarding_page.dart]: zasto imamo dva onboarding pagea?

**Review by matejthetree — CHANGES_REQUESTED** [review]

File: lib/ui/page/code_onboarding_page.dart (hunk 8 above)
Comment (Tinjomar) [reply lib/ui/page/code_onboarding_page.dart]: slučajno

**Review by Tinjomar — COMMENTED** [review]
resolved

File: lib/ui/page/onboarding_page.dart [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+              height: 310,
+              decoration: BoxDecoration(
+                  color: Colors.white,
+                  image: DecorationImage(
+                      image: NetworkImage('https://placekitten.com/200/300'))))),
+    );
+  }
+
+  void onDonePress() {  // Done button is START button
+    Navigator.pushNamed(context, 'login');
```
Comment (matejthetree) [inline lib/ui/page/onboarding_page.dart:57]: `// todo add login route once finished`

Line 52 in .../..../.../ file add push to login route

**Review by matejthetree — APPROVED** [review]
little changes and FORMATTING

**Comment by Noamagnus** [comment]
Review done

**Review by MrgaCRO — APPROVED** [review]

**Review by Noamagnus — APPROVED** [review]
Approved

**Review by matejthetree — APPROVED** [review]

**Comment by matejthetree** [comment]
rebase i merge

**Pull request merged**